GiEnJul/ClientApp/src/components/admin/overview/OverviewMacro.tsx

```diff
@@ -72,11 +68,11 @@ function OverviewMacro() {
       .post(
         "admin/",
         JSON.stringify({
-          GiverRowKey: selectedConnection.current.giverRowKey,
-          GiverPartitionKey: selectedConnection.current.giverPartitionKey,
-          RecipientRowKey: selectedConnection.current.recipientRowKey,
+          GiverRowKey: selectedConnection.current.giver?.rowKey,
```

Comment:
Is it possible to just stringify it with {selectedConnection.current.GiverType, ...RecipientType} or {selectedConnection}?

Reply:
Oh that was supposed to be a "comment", oh well

Reply:
I tried to do it like that but it has to be declared somehow. Don't know why.